lambdas/src/bnas/get-bnas.rs

```diff
@@ -0,0 +1,50 @@
+use dotenv::dotenv;
+use entity::bna;
+use lambda_http::{run, service_fn, Body, Error, IntoResponse, Request, RequestExt, Response};
+use lambdas::pagination_parameters;
+use sea_orm::{prelude::Uuid, Database, EntityTrait, PaginatorTrait};
+use serde_json::json;
+use std::env;
+
+async fn function_handler(event: Request) -> Result<Response<Body>, Error> {
+    dotenv().ok();
+
+    // Set the database connection.
+    let database_url = env::var("DATABASE_URL")?;
+    let db = Database::connect(database_url).await?;
```

Comment:
do you have to release/close the connection? one connection per lambda and when the run finishes the db connection is put back, right?